components/tests.py

```diff
@@ -757,3 +758,29 @@ def test_add_control_description(self):
             if implemented.get("control-id") == test_control_id:
                 self.assertEqual(implemented.get("description"), test_description)
                 self.assertEqual(implemented.get("control-id"), test_control_id)
+
+
+class LoadComponentsTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        call_command("load_catalog", load_standard_catalogs=True)
+
+    def test_load_test_components(self):
+        call_command("load_components")
+
+        expected_components = [
+            "Amazon Web Services",
+            "Blueprint", "Django",
+            "Identity Management",
+            "OCISO Inheritable Controls",
+            "Splunk",
+            "Tenable Nessus",
+            "Trend Micro Deep Security"
+        ]
+
+        queryset = Component.objects.order_by("title")
+
+        self.assertEqual(queryset.count(), 8)
+        self.assertEqual(expected_components, [item.title for item in queryset])
+        self.assertTrue(all(item.type == "software" for item in queryset))
```

Comment:
OCISO type may change to `policy` going forward. Just a heads up.

Reply:
Only if we change the value in the file in this repo though ;). If we want to, we can of course.